lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -41,7 +41,7 @@ sub new {
             die "$@ \n" if $@;
             $self->{db} = "$params->{db}"->new();
         };
-        die "$@ \n" if $@;
+        die "Failed to initialize the SQLite database backend module \n" if $@;
```

Comment:
I expect some general message through the API and the detailed message to a log file. Is it always SQLite problem in this case?

Reply:
This test is only true if the full perm module is given as input. This only happens during testing from the backend unit test scripts.